modelconverter/hub/__main__.py

```diff
@@ -0,0 +1,688 @@
+import logging
+from pathlib import Path
+from typing import Any, Dict, List, Literal, Optional
+from urllib.parse import unquote, urlparse
+
+import requests
+import typer
+from luxonis_ml.nn_archive import is_nn_archive
+from rich import print
+
+from modelconverter.cli import (
+    ArchitectureIDOption,
+    CommitHashOption,
+    CompressionLevelOption,
+    DescriptionOption,
+    DescriptionShortOption,
+    DomainOption,
+    FilterPublicEntityByTeamIDOption,
+    HashOption,
+    HubVersionOption,
+    HubVersionOptionRequired,
+    IdentifierArgument,
+    IsPublicOption,
+    JSONOption,
+    LicenseTypeOption,
+    LicenseTypeOptionRequired,
+    LimitOption,
+    LinksOption,
+    LuxonisOnlyOption,
+    ModelClass,
+    ModelIDOption,
+    ModelIDOptionRequired,
+    ModelType,
+    ModelTypeOption,
+    ModelVersionIDOption,
+    ModelVersionIDOptionRequired,
+    NameArgument,
+    NameOption,
+    OptimizationLevelOption,
+    OptsArgument,
+    Order,
+    OrderOption,
+    OutputDirOption,
+    ParentIDOption,
+    PathOptionRequired,
+    PlatformsOption,
+    ProjectIDOption,
+    Quantization,
+    QuantizationOption,
+    RepositoryUrlOption,
+    SilentOption,
+    SlugOption,
+    SortOption,
+    StatusOption,
+    TagsOption,
+    TargetArgument,
+    TargetPrecisionOption,
+    TasksOption,
+    TeamIDOption,
+    UserIDOption,
+    VariantSlugOption,
+    VersionOption,
+    get_configs,
+    get_resource_id,
+    get_target_specific_options,
+    get_version_name,
+    get_version_number,
+    hub_ls,
+    print_hub_resource_info,
+    request_info,
+    wait_for_export,
+)
+from modelconverter.cli.types import License, TargetPrecision
+from modelconverter.utils.types import Target
+
+from .hub_requests import Request
+
+logger = logging.getLogger(__name__)
+app = typer.Typer(
+    help="Hub CLI",
+    add_completion=False,
+    rich_markup_mode="markdown",
+)
+
+model = typer.Typer(
+    help="Models Interactions",
+    add_completion=False,
+    rich_markup_mode="markdown",
+)
+
+version = typer.Typer(
+    help="Hub Versions Interactions",
+    add_completion=False,
+    rich_markup_mode="markdown",
+)
+
+instance = typer.Typer(
+    help="Hub Instances Interactions",
+    add_completion=False,
+    rich_markup_mode="markdown",
+)
+
+app.add_typer(model, name="model", help="Models Interactions")
+app.add_typer(version, name="version", help="Hub Versions Interactions")
+app.add_typer(instance, name="instance", help="Hub Instances Interactions")
+
+
+@model.command(name="ls")
+def model_ls(
+    team_id: TeamIDOption = None,
+    tasks: TasksOption = None,
+    user_id: UserIDOption = None,
+    license_type: LicenseTypeOption = None,
+    is_public: IsPublicOption = True,
```

Comment:
Would have `is_public = None` by default. This should then only show models belonging to a particular team and not all models in zoo as well. This makes sense from conversion perspective because we can only convert models belonging to the team.